tox.ini

```diff
@@ -1,16 +1,23 @@
 [tox]
-envlist = lint, test
+envlist = flake8, pylint, test
```

Comment:
What do you mean by block?

Reply:
I'm used to the CI flow aborting if Tox fails. We'll be hard pressed to get Pylint to pass every time. I suppose you could just run things via `-e` in travis, but I feel like Travis should just run `tox`.

Reply:
(answered below)

Reply:
For now let's ignore pylint's results. If later on we want to stop doing that, we'll probably need to tell it to ignore some errors otherwise our builds will fail all the time.

Reply:
I was just saying that you could ignore it by not adding it to this line. It will just slow things down to run something we're not using. You can always invoke it will `tox -e pylint` just like we do with `tox -e cleanup`.

Reply:
Good point. My thinking was to run it by default so we would see what `pylint` says, and I'm ignoring the results to not break CI.

Reply:
It's a choice between giving people visibility when they run it locally or speeding up the CI flow and cleaning up the CI logs. Your choice.

Reply:
Let's try and see how it goes. We can always change it later on.

Reply:
Sounds great :)